src/disease.py

```diff
@@ -1,10 +1,28 @@
 # coding: utf-8
+import logging
 
+from .exceptions import GameCrisisException
 
-class Disease:
+
+class NoHealthException(GameCrisisException):
     def __init__(self, colour):
         self.colour = colour
+
+    def __str__(self):
+        return f'Public health resistance vs {self.colour} disease is no more! Pandemics!'
```

Comment:
I suggest "Public health resistance to {self.colour} disease is over! Pandemic!"